### DISCUSSION TITLE
May i know where can i get access for my custom module

### DISCUSSION THREAD

**Discussion by lemankk** [directus/directus#2853] [Archive]
I tried to follow the page to create demo module from https://docs.directus.io/extensions/modules.html#files-structure
 but i cannot find resource about how to access it and link to system. May i know anyone knows about it?

**Comment by Ncturn** [comment]
Hi @lemankk  did you find how access to a custom module, i have the same issue i create one module with this info:
https://docs.directus.io/guides/modules/
But i cant see the module or get access to it.

**Comment by hao-hao-hao** [comment]
rename the dist folder to a name such as new-module. Move the whole folder over to the directus extesions/modules
